libs/expo/betterangels/src/lib/screens/Client/Profile/ContactInfo.tsx

```diff
@@ -79,7 +85,7 @@ export default function ContactInfo(props: IProfileSectionProps) {
       mb="xs"
       title="Contact Info"
     >
-      {isContactInfo && (
+      {isContactInfo && hasContent && (
```

Comment:
separate, and more of a product question, but when a user clicks a ContactInfo link and nothing show, they may think it's just broken. Maybe should show a comment that nothing to see here.

Reply:
I agree. This applies to all profile sections